compiler-rt/lib/scudo/standalone/mem_map_base.h

```diff
@@ -93,9 +93,11 @@ template <class Derived, typename MemMapTy> class ReservedMemory {
   constexpr ReservedMemory() = default;
 
   // Reserve a chunk of memory at a suggested address.
-  bool create(uptr Addr, uptr Size, const char *Name, uptr Flags = 0) {
+  bool create(uptr Addr, uptr Size, const char *Name, uptr Flags = 0,
+              uptr Alignment = getPageSizeCached()) {
```

Comment:
Would it be better to change the alignment to be something like AlignmentPages and have the default be 1?

A lot of the places are checking that alignment is equal to getPageSizeCached() which is free on Android (most of the time), but is not as free on devices that don't have a hard-coded page size.

Reply:
This is a good idea. Done.

BTW, I haven't rebased so I haven't changed to use `getPageSizeLogCached` accordingly. Will do it later.